ui/src/pages/Formation/formation.css

```diff
@@ -0,0 +1,3 @@
+strong {
+  background: #f9f8f6;
+}
```

Comment:
C'est mieux de ne plus faire de fichier css à côté et d'utiliser uniquement le style CSS in JS

Reply:
ok c'est noté